### ISSUE TITLE
nvda does not announce the drive name in focus when pressing windows + e and nvda version 22,621,160

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#13804]
### Steps to reproduce:
I know that this feature in Windows is in testing and that improvements and troubleshooting can be done.
But please investigate whether nv access should already do something to adapt nvda to the new feature.
download windows 11 beta channel update
windows 11 version 22.621.160
in addition to having this update if it is not contemplated with the test it may be necessary to enable the new features manually
Now open explorer using the shortcut windows + and
I used the shortcut in the Brazilian Portuguese version of windows, I hope this shortcut is in other languages.
Now by default windows opens the file explorer to a view called computer.
### Actual behavior:
nvda only advertises
This Computer
ShellView
Shell Folder View
panel
### Expected behavior:
Este Computador
Modo de Exibição de Itens  lista
Unidades de Disco Rígido  grupo  expandido
Acer (C:)  não selecionado  1 de 2

please abstract the unit name and the amount of items in the list
#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
nvda.exe, NVDA alpha-25587,fb1ce94d

#### Windows version:
22.621.160
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes
log
[computador_621.160.txt](https://github.com/nvaccess/nvda/files/8917973/computador_621.160.txt)

**Comment by Adriani90** [comment]
@fernando-jose-silva I cannot reproduce this in Windows 11 22 H2 update with NVDA last alpha version. For me, when i press windows+e, NVDA reports also the drive name "OS (C)".
Can you still reproduce this issue?

**Comment by fernando-jose-silva** [comment]
I don't have this problem anymore, thank you very much

Em 01/04/2023 20:28, Adriani90 escreveu:

**Comment by Adriani90** [comment]
Closing as works for me.
